### ISSUE TITLE
Xcode doesn't support plugin xcodecolor anymore... what to do?

### ISSUE THREAD

**Issue by rlam3** [honghaoz/Loggerithm#15]
Xcode doesn't support plugin xcodecolor anymore... what to do?

**Comment by honghaoz** [comment]
There's no way to support console text color under Xcode8, one way to use the plugin is mentioned in this [issue](https://github.com/alcatraz/Alcatraz/issues/475#issuecomment-225868817).

Or just wait for future Xcode to support more advanced Xcode extensions.  ¯\_(ツ)_/¯

**Comment by rlam3** [comment]
Aww man that really is  bummer.... Are there alternatives? I've been looking into SwiftyBeaver lately. What do you think of it @honghaoz ?

**Comment by honghaoz** [comment]
That's a good choice. You see the 💜 is an alternative way for coloured logging.
I'm also using [XCGLogger](https://github.com/DaveWoodCom/XCGLogger).

**Comment by rlam3** [comment]
@honghaoz But XCGLogger doesn't support the coloring as well. It just outputs it as a string in xcode right?

**Comment by honghaoz** [comment]
@rlam3 Yes, with official Xcode, you cannot get coloured log strings ¯\_(ツ)_/¯
